worker/build.sbt

```diff
@@ -18,21 +18,22 @@ libraryDependencies ++= Seq(
   "org.joda" % "joda-convert" % "1.7",
   "com.typesafe" % "config" % "1.0.0",
   "com.datadoghq" % "java-dogstatsd-client" % "4.1.0",
-  "org.scala-lang.modules" %% "scala-collection-compat" % "2.1.6"
+  "com.google.code" % "simplelrucache " % "0.1",
```

Comment:
I don't think this package is maintained anymore. The last update was 10 years ago. Could we use a different library that is better maintained. 

You could probably even use [java.util.LinkedHashMap](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/util/LinkedHashMap.html) configured to use access order, and override `removeEldestEntry` to remove the oldest if the collection reaches a certain size.

Reply:
Using a retry system would remove the need for this package

Reply:
or you could use [caffeine](https://github.com/ben-manes/caffeine) that seems to be a pretty standard solution for caching in java

Reply:
I'll just keep with the retry system for now